app/services/player-db-service.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const DbService = require('./db-service');
+
+class PlayerDbService {
+    constructor() {
+        this._dbService = new DbService();
+    }
+
+    createNew(playerId, playerName) {
+        try {
+            this._dbService.performQuery(
+                'INSERT INTO player (id, name, high_score) VALUES($1, $2, $3)',
```

Comment:
Se #20, vi trenger følgende informasjon om spilleren:
- id
- name
- current_score
- highest_score

Reply:
Trenger vi å lagre/hente current_score? Jeg har lagt til kolonne for det i databasen just in case, men tenkte i etterkant at current_score kanskje ikke er nødvendig fordi den er bare midlertidig uansett. For når man starter nytt spill vil jo current score være 0?